server/server.go

```diff
@@ -894,11 +894,9 @@ func (server HTTPServer) buildReportEndpointResponse(
 	)
 	log.Info().Msgf("Cluster ID: %v; visible rules %d, no content rules %d, disabled rules %d", clusterID, len(visibleRules), noContentRulesCnt, disabledRulesCnt)
 
-	if err != nil {
-		if _, ok := err.(*content.RuleContentDirectoryTimeoutError); ok {
-			handleServerError(writer, err)
-			return nil, 0, err
-		}
+	if _, ok := err.(*content.RuleContentDirectoryTimeoutError); ok {
```

Comment:
does not it throw `nil pointer`?

Reply:
no, `ok` will be false in case of nil https://play.golang.com/p/OOj8m7WJobe , in fact it was suggested by my linter to remove the nil check https://staticcheck.io/docs/checks#S1020

Reply:
cool, TY